core/arch/asm_defines.asm

```diff
@@ -418,6 +418,9 @@ ASSUME fs:_DATA @N@\
 # define REG_R29  x29
 # define REG_R30  x30
 # define REG_R31  x31
+# define REG_A0   x10
+# define REG_A1   x11
+# define REG_A2   x12
```

Comment:
These macros seem unused.

Reply:
These values are used from assembly macros expansion of CALLC1 core/arch/asm_defines.asm:1000